### PR TITLE
Update Facebook SDK

### PR DESCRIPTION
**Pull request by zambodaniel**
I've updated the facebook sdk to v13 because the v12 is deprecated and throwing excepions on data push.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
         "roave/security-advisories": "dev-latest",
         "setono/code-quality-pack": "^2.1.2",
         "setono/sylius-behat-pack": "^0.1",
-        "sylius/sylius": "~1.7.0",
+        "sylius/sylius": "^1.10",
```
Comment (loevgaard) [inline composer.json]: Do not update this unless you make sure the 1.10 application works :)

File: composer.json [hunk 2]
```diff
@@ -76,7 +76,12 @@
         ]
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "symfony/thanks": false,
+            "dealerdirect/phpcodesniffer-composer-installer": true,
```
Comment (loevgaard) [inline composer.json]:
```suggestion
            "dealerdirect/phpcodesniffer-composer-installer": false,
```

File: composer.json (hunk 1 above)
Comment (zambodaniel) [reply composer.json]: I'm on 1.10, its working.

Comment (loevgaard) [reply composer.json]: I meant the test application that is bundled together with the plugin :)

Comment (loevgaard) [reply composer.json]: The easiest is if you just revert this change :)

**Comment by loevgaard** [comment]
This change was added by dependabot here: https://github.com/Setono/SyliusFacebookPlugin/pull/60

**Pull request closed without merging**